lib/server/ServerRenderer.js

```diff
@@ -9,7 +9,14 @@ var ServerRenderer = {
     render: function(layout, view, environmentConfig, clientAppRequirePath, serverRequest) {
         var title = view.getTitle ? view.getTitle() : null;
         var metatags = view.getMetatags ? view.getMetatags() : null;
-        var appRoot = environmentConfig ? environmentConfig.appRoot : "";
+        var config = environmentConfig || {};
+        var appRoot = config.appRoot || "";
+        var clientAppBootstrapFunctionName = null;
```

Comment:
another way to do lines 14-18 and the check on line 83 is to set pass config.clientAppBootstrapFunctionName to makeClientAppStartScript and have line 83 ask the question "typeof clientAppBoostrapFunctionName !== 'string'"